pynicotine/gtkgui/chatrooms.py

```diff
@@ -266,7 +271,7 @@ def join_room(self, msg):
             # Did not auto-join room, switch to tab
             core.chatrooms.show_room(msg.room)
 
-        if msg.room == core.chatrooms.GLOBAL_ROOM_NAME:
+        if is_global:
             self.roomlist.toggle_public_feed(True)
```

Comment:
The activation behavior is inconsistent.

The first time it appears in the background (without changing focus). Thereafter when it is disabled/re-enabled for a second time then it grabs the tab focus.